data/channels.csv

```diff
@@ -4782,6 +4785,7 @@ ClubTV.cl,Club TV,,,,CL,CL-BI,Santa Juana,c/CL,spa,,FALSE,,,,https://www.clubtv.
 CMAC1.us,CMAC 1,,,,US,US-CA,Fresno,s/US-CA,eng,general,FALSE,,,,https://cmac.tv/public/,https://i.imgur.com/PYhdQy9.png
 CMAC2.us,CMAC 2,,,,US,US-CA,Fresno,s/US-CA,eng,education,FALSE,,,,https://cmac.tv/edu/,https://i.imgur.com/O0sYOED.png
 CMAC3.us,CMAC 3,,,,US,US-CA,Fresno,s/US-CA,eng,legislative,FALSE,,,,https://cmac.tv/gov/,https://i.imgur.com/ToZzguz.png
+CMalayalamTV.ca,C Malayalam TV,,,,CA,,,c/CA,mal,general,FALSE,,,,https://www.canadianmalayalamtv.com/,https://i.imgur.com/WPksZUC.png

```

Comment:
```suggestion
CMalayalamTV.ca,C Malayalam TV,,,,CA,,,c/CA,mal,general,FALSE,,,,https://www.canadianmalayalamtv.com/,https://i.imgur.com/rpg0bQA.png
```
Removed the transparent background around the logo.

Reply:
It already had transparency!